scripts/combine/setupCombine.py

```diff
@@ -47,7 +46,7 @@ def make_parser(parser=None):
     parser.add_argument("-v", "--verbose", type=int, default=3, choices=[0,1,2,3,4],
                         help="Set verbosity level with logging, the larger the more verbose")
     parser.add_argument("--noColorLogger", action="store_true", help="Do not use logging with colors")
-    parser.add_argument("--hdf5", action="store_true", help="Write out datacard in hdf5")
+    parser.add_argument("--root", action="store_true", help="Write out datacard in txt/root format")
```

Comment:
Looks like at least some of the problems in the CI are again related to this change (we also backed this out last time to speed up merging of the N^(3+0)LL uncertainties in the first place)